articles/quickstart/backend/_includes/_call_api.md

```diff
@@ -1,6 +1,6 @@
 ## Make a Call to Your API
 
-You can now make calls to your secure API by providing the `access_token` as an `Authorization` header in your requests.
```

Comment:
Add a section here saying
'
To make calls to your API, you need an access token. For testing purposes, you can obtain it from the Test tab in your [API settings](${manage_url}/#/apis):
  
![Obtain a JWT](/media/articles/server-apis/aspnet-core-webapi/request-access-token.png)
'